docs/OPERATIONS.md

```diff
@@ -0,0 +1,101 @@
+# Deploy a Strategy / Vault for V2
+
+NOTE: This repo is encouraged to create multiple scripts for governance and dev multisig execution of complex transactions
+https://github.com/iearn-finance/chief-multisig-officer
+
+**IMPORTANT**: you should **NOT** create a new release with a test vault
+
+## Before deploying
+- [ ] Coordinate with Core Dev strategist for getting a review in board https://github.com/orgs/iearn-finance/projects/5
+- [ ] Peer review completed by:
+    - yearn.rocks: at least 2 strategists
+    - Experimental: at least 1 strategist + 1 core dev in board 
+- [ ] Check if want token has a deploy vault already (>=v0.3.0) and coordinate to use that first if possible.
+- [ ] (only if a new vault is needed) Deploy using the registry.
+    - `registry.newExperimentalVault(want, you, multisig, treasury, "", "")`
+        - args: token, gov, guardian, rewards, name, symbol
+        - multisig (strategists) = '0x16388463d60FFE0661Cf7F1f31a7D658aC790ff7'
+        - gov for testing = '0x846e211e8ba920b353fb717631c015cf04061cc9' (dev.ychad.eth) 
+        - treasury = '0x93A62dA5a14C80f265DAbC077fCEE437B1a0Efde'
+        (treasury.ychad.eth)
+    - [ ] Coordinate with core devs proper cap limit for new vault and other settings
```

Comment:
Are caps arbitrary? If so, maybe mention what the criteria to decide on limits are. 

If possible, would be even better to have some kind of table for what types of limits we set, and how to progress to the next stage.

Reply:
Caps are more or less arbitrary. Nevertheless, we need this you are suggesting. Will add. Nice catch.

Reply:
Caps added.